.github/workflows/maven.yml

```diff
@@ -3,25 +3,25 @@
 
 name: Java CI with Maven
 
-on: [push, pull_request]
+on: [ push, pull_request ]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
-      with:
-        show-progress: false
-        # needed for the git-commit-id-plugin to work
-        fetch-depth: 0
-    - name: Set up JDK 8
-      uses: actions/setup-java@v3
-      with:
-        java-version: '8'
-        distribution: 'temurin'
-    - name: Maven Install
-      run: mvn install -B -V -DskipTests -Dair.check.skip-all
-    - name: Maven Tests
-      run: mvn install -B -P ci
+      - uses: actions/checkout@v4
+        with:
+          show-progress: false
+          # needed for the git-commit-id-plugin to work
+          fetch-depth: 0
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Maven Install
+        run: mvn install -B -V -DskipTests -Dair.check.skip-all
+      - name: Maven Tests
+        run: mvn install -B -P ci
```

Comment:
Can we remove whitespace changes? This file doesn't seem to have actually been touched